pydantic/generics.py

```diff
@@ -113,4 +123,34 @@ def _parameterize_generic_field(field_type: Type[Any], typevars_map: Dict[TypeVa
 
 
 def _is_typevar(v: Any) -> bool:
```

Comment:
I propose changing the name of this function to something better, since it's no longer doing a simple check to see if something is a type variable. Maybe something like `_is_generic` or `_has_typevar`?

Reply:
Good idea, I renamed it to `_has_typevar` as suggested